modules/tensor_mechanics/test/tests/torque_reaction/disp_about_axis_axial_motion.i

```diff
@@ -132,35 +43,48 @@
   [./top_y]
     type = DisplacementAboutAxis
     boundary = top
-    function = rampConstant
+    function = 't'
     angle_units = degrees
     axis_origin = '0. 0. 0.'
     axis_direction = '0. 0. 1.'
     component = 1
     variable = disp_y
   [../]
-
-# Because want to keep the rotation fixed about the z axis,
-# do not apply a DisplacementAboutAxis BC on the disp_z
```

Comment:
I'd like to keep these two lines of comments in the input file --this very explanation actually helped me out a bunch when I was setting up the torsion tube simulation

Reply:
Sure, I can add them back.  I thought they were confusing, myself!

Reply:
If there's a better way to word the comments, please change it to make it less confusing. I had found the 'captain obvious' comments helpful when I was trying to figure out if I should or should not apply a `disp_z` boundary condition

Reply:
Note that I revised this comment and moved it above this set of BCs. It actually is OK to specify those in the z direction, but it would give the same result in this case since there is no deformation. It would be equivalent to applying a DirichletBC in the z direction.